src/SprykerEco/Zed/AfterPay/Communication/Plugin/Oms/Command/CancelPlugin.php

```diff
@@ -23,6 +23,8 @@
 class CancelPlugin extends AbstractPlugin implements CommandByOrderInterface
 {
     /**
+     * {@inheritDoc}
+     *
```

Comment:
* - Sends "void" request to AfterPay gateway, to cancel payment for a specific order item, before payment is captured
 * - If it is the last item cancellation request for given order, cancels also full expense amount.
 * - Saves the transaction result in DB and updates payment with new total cancelled amount.